VS_Project/Pokerapp/Pokerapp/spelers-uitschakelen.cshtml

```diff
@@ -0,0 +1,93 @@
+﻿@using WebMatrix.Data
+
+@{
+    string connectionString = @"Data Source=(LocalDB)\MSSQLLocalDB;AttachDbFilename=|DataDirectory|\Database.mdf;Integrated Security=True";
+    string provider = "System.Data.SqlClient";
+    Database db = Database.OpenConnectionString(connectionString, provider);
+
+    //Insert Nick into Players
+    string name = "Nick";
+    int room_code = 1234567;
+    int tbl_nr = 2;
+    //int rebuy = 1;
+    //string insert = "INSERT INTO Players (Name, Room_Code, Tbl_nr, Rebuy) VALUES (@0, @1, @2, @3)";
+    //db.Execute(insert, name, room_code, tbl_nr, rebuy);
+
+    int maxrebuy = 3;
+    string player = (string)db.QueryValue("SELECT Name FROM Players WHERE Name = @0 AND Room_Code = @1 AND Tbl_nr = @2", name, room_code, tbl_nr);
+    int rebuyvalue = (int) db.QueryValue("SELECT Rebuy FROM Players WHERE Name = @0 AND Room_Code = @1", name, room_code);
+
+    if (IsPost && !Request.Form["rebuy"].IsEmpty() && rebuyvalue < maxrebuy)
+    {
+        var rebuyplayer = db.Execute("UPDATE Players SET Rebuy = @0, Tbl_nr = @1 WHERE Name = @2 AND Room_Code = @3", rebuyvalue + 1, tbl_nr, name, room_code);
+    }
+
+    if (IsPost && !Request.Form["rebuy"].IsEmpty() && rebuyvalue == maxrebuy - 1 || rebuyvalue == maxrebuy)
+    {
+    <!-- Modal -->
```

Comment:
Als ik mij goed herrinner is dit niet hoe een modal werkt. Een modal blijft onzichtbaar tot je zijn trigger activeerd dus is het niet nodig om hem achter een if statement te gooien. Je hoeft enkel zijn trigger te activeren.